### ISSUE TITLE
should change the uiTheme to light theme ?

### ISSUE THREAD

**Issue by Youzhigang** [slack-theme/visual-studio-code#9]
https://github.com/Youzhigang/visual-studio-code/blob/8cb88e276e73524c0c6691a1c67c81a5e49d859e/package.json#L32
this uiTheme is dark type, and the editor.backgroud is light, so the mouse pointer color is light. it's hard to find the mouse pointer.

**Comment by Youzhigang** [comment]
I like this vscode theme very much.

**Comment by felipemendes** [comment]
Hi @Youzhigang. Thank you for your contribution. Glad to know you appreciate it.
If uiTheme is set to light type some buttons are display as dark gray color. There is a PR ( #7 ) open to fixed that. Unfortunately, I didn't figure out how to fix it :(

**Comment by Youzhigang** [comment]
project/package.json/contributes/themes/uiTheme, 
![image](https://user-images.githubusercontent.com/18167983/59398773-0d6f1180-8dc4-11e9-86e6-bf0759344bcb.png)
is it right?

**Comment by felipemendes** [comment]
Yes, it is right.
This is how each one look like on Aubergine color:

vs
![vs](https://user-images.githubusercontent.com/3712089/59399811-c05f5100-8d6b-11e9-9bc3-f5aad5cebee6.png)

vs-dark
![vs-dark](https://user-images.githubusercontent.com/3712089/59399814-c2c1ab00-8d6b-11e9-93e1-2c73f9787d7f.png)

hc-black
![hc-black](https://user-images.githubusercontent.com/3712089/59399817-c48b6e80-8d6b-11e9-897b-b239b893191c.png)


If set to light theme 'vs' basically all icons above dark colors won't be visible. It should probably be able to adapt 'hc-black' and fix this issue. I will try it later.

**Comment by shacker** [comment]
Separate but related question (using it in the regular editor, not in Jupyter): I love the Ochin theme, but the mouse cursor is impossible to see. I know this is not a problem with all light themes - I can see the mouse cursor fine in other light themes. I am trying to customize it via 

```
"editorCursor.foreground": "#a33e3e",
```

which seems intuitive, but doesn't have any effect. I've exported the Ochin theme and studied it but can't seem to figure out which parameter is tweaked to make the mouse cursor invisible. So: A) Do you consider this a bug in the theme? and B) what can I tweak to override and make it visible again?

Thanks.

**Comment by shacker** [comment]
I notice the Hoth theme doesn't have this problem. But comparing them, still cannot figure out what's different about them to change the cursor color.

**Comment by pkazmier** [comment]
@shacker There is nothing that we, as users of the theme, can do to control the appearance of a few elements. The pointer color is one of those items. Another item is preformatted text in the hover widget. In my opinion, it's too dark for my taste:

![image](https://user-images.githubusercontent.com/747855/78058755-635e7700-734e-11ea-9d02-3653cf4c2a2f.png)

The Hoth version of the theme does not suffer from this problem:

![image](https://user-images.githubusercontent.com/747855/78058831-7cffbe80-734e-11ea-802c-1d11c5beaa3e.png)

I'm sure there are other differences, but the pointer color and the above are not controlled by anything in the actual theme file, which is a bummer. The only way workaround I've found is to edit the theme's `package.json` in your VSCode extensions directory. On my system, I edited:

`~/.vscode/extensions/felipe-mendes.slack-theme-1.9.11/package.json` 

You'll find a section towards the bottom, with the following:

```
{
    "label": "Slack Theme Aubergine - NEW COLOR",
    "uiTheme": "vs-dark",
    "path": "./themes/aubergine-new.json"
},
```

Change the `uiTheme` from `vs-dark` to `vs`:

```
{
    "label": "Slack Theme Aubergine - NEW COLOR",
    "uiTheme": "vs",
    "path": "./themes/aubergine-new.json"
},
```

Save. Reload your VSCode session. After reloading, it'll then complain the extensions have changed, so you'll need to reload again. Finally, you'll have both items fixed. The screenshot below doesn't show the pointer, but trust me, the pointer is now visible again:

![image](https://user-images.githubusercontent.com/747855/78059701-f1872d00-734f-11ea-8e46-f56beeccbb16.png)

I'm sure there are some other things UI elements that will look different, but I'm not sure of the exhaustive list. I did notice that the selected file in the file tree on the left is now harder to read, so I'm not sure if that can be adjusted. I need to look around more myself.
